File: Praktikum I Modul 5/Modul 5/Python/PRAK504-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# fungsi
def reverse(nilai):
    reverseNilai = 0

    # perulangan
    while (nilai != 0):
        sisa_bagi = nilai % 10
        reverseNilai = reverseNilai * 10 + sisa_bagi
        nilai //= 10

    return reverseNilai

# input
a, b = input().split()

# konversi ke tipe data integer
a = int(a)
b = int(b)

a = reverse(a)
b = reverse(b)

c = a + b
# output
print(reverse(c))

File: Praktikum I Modul 5/Modul 5/C/PRAK504-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

// fungsi
int reverse(int nilai){
    // deklarasi variabel
    int reverseNilai = 0, sisa_bagi;
    
    // perulangan
    while (nilai != 0){
        sisa_bagi = nilai % 10;
        reverseNilai = reverseNilai * 10 + sisa_bagi;
        nilai /= 10;
    }
    return reverseNilai;
}

int main(){
    // deklarasi variabel
    int A, B, C;

    // input A dan B
    scanf("%d %d", &A, &B);

    A = reverse(A);
    B = reverse(B);
    C = A + B;
    
    // output
    printf("%d\n", reverse(C));

    return 0;
}